### DISCUSSION TITLE
Section profile

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#392] [Q&A] [answered]
Would it be possible to use different routing profiles for different route sections?
That happened in Kurviger.
E.g.: The first part of a tour (should lead as quickly as possible) to the actual destination area.
The second part (as curvy as possible) around there...
Not only would the actual tour have to be saved, but also the calculation method...Or many waypoints...

**Comment by devemux86** [comment, marked as answer]
@Leeb64 thanks for the suggestion.

This is already planned, please see:
- https://github.com/devemux86/cruiser/discussions/142
